portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,42 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+import java.util.ArrayList;
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  public ArrayList<String> comments = new ArrayList<String>();
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    final Gson gson = new Gson();
+    String jsonComments = gson.toJson(comments);
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Sydney!</h1>");
+    response.getWriter().println(jsonComments);
+    
+    /*final Gson gson = new Gson();
+    ArrayList<String> comments = new ArrayList<>();
+
+    comments.add("Great Job!");
+    comments.add("Keep up the good work!");
+    comments.add("Awesome!"); */
```

Comment:
Usually we don't submit PRs with commented code (comments) in it since it has no functions, and Java compiler ignores them any way. Can you remove these commented code?

Reply:
Or looks like you need to uncomment them? Since it will be used in `doPost(...)` function?